rutil/Data.hxx

```diff
@@ -181,6 +184,13 @@ class Data
       */
       explicit Data(const std::string& str);
 
+#if RESIP_CPP_STANDARD >= 201703L
+      /**
+        Creates a data with the contents of the string_view.
+      */
+      explicit Data(const std::string_view sv);
```

Comment:
Please add unit tests for the extended functionality to [testData.cxx](https://github.com/resiprocate/resiprocate/blob/master/rutil/test/testData.cxx)

Reply:
Thanks, added.